schema/artwork/index.js

```diff
@@ -43,6 +43,10 @@ const is_inquireable = ({ inquireable, acquireable }) => {
   return (inquireable && !acquireable);
 };
 
+const is_price_range = ({ price }) => {
```

Comment:
Good point! I initially only passed in the price instead of the entire artwork obj but saw the pattern above and switched it. I see now that there are two params above so the entire artwork makes sense in that case. Gonna make the change now.